docs/guide/etl.rst

```diff
@@ -564,9 +564,10 @@ Extension
 ---------
```

Comment:
I've only seen that once before a year or two ago, and I'm not sure what it is. It's valid RST, because the time I saw it before it was flagging JSON. Maybe it's character encoding? Lines ending with `\n\r`?